examples/packages/type-graphql/package.json

```diff
@@ -19,9 +19,15 @@
   },
   "dependencies": {
     "@apollo/server": "4.7.5",
-    "type-graphql": "1.1.1"
+    "@graphql-authz/apollo-server-plugin": "3.1.1",
+    "@graphql-authz/core": "1.3.1",
+    "type-graphql": "1.1.1",
+    "graphql": "15.3.0",
+    "reflect-metadata": "0.1.13"
   },
   "devDependencies": {
-    "typescript": "4.9.3"
+    "typescript": "4.9.3",
```

Comment:
if `examples` is a part of common workspace, then packages from it shouldn't have own devDependencies. instead, dev dependencies should be listed in root package json

Reply:
It's added here, so this package could be independent and could be moved / copy-pasted anywhere without any changes required.